Actions/DetermineBuildProject/DetermineBuildProject.ps1

```diff
@@ -0,0 +1,51 @@
+﻿Param(
```

Comment:
Is there a way to add tests for the new action?

Reply:
It is thoroughly tested in the end 2 end test - but as it is downloading artifacts and checking repos - it will be hard to make a unit test which provides any value really